Makefile.js

```diff
@@ -70,6 +70,9 @@ target.lint = function() {
 
 	var code = 0;
 
+	echo('Compiling TypeScript code');
+	code += nodeCLI.exec('./node_modules/.bin/tsc').code;
```

Comment:
What do you think about moving this to a new `target.build` function specifically for this, and then add a "build" script to package.json that just runs `node Makefiles.js build`? I'm thinking of our release process in particular 🤔

Reply:
Good idea, applied the changes!